src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -268,11 +268,16 @@ local g_tLuaKeywords = {
     ["while"] = true,
 }
 
+local serialize_infinity = math.huge
 local function serialize_impl(t, tracking, indent, opts)
     local sType = type(t)
     if sType == "table" then
         if tracking[t] ~= nil then
-            error("Cannot serialize table with recursive entries", 0)
+            if tracking[t] == t then
```

Comment:
Maybe `true`/`false` here instead, where `true` marks us as currently visiting. Just feels a cleaner tri-state, and avoids us having to worry about `__eq`.

Reply:
That would indeed be smarter way to do it. 😊